polkadot/runtime/parachains/src/runtime_api_impl/v7.rs

```diff
@@ -92,16 +93,41 @@ pub fn availability_cores<T: initializer::Config>() -> Vec<CoreState<T::Hash, Bl
 		.enumerate()
 		.map(|(i, core)| match core {
 			CoreOccupied::Paras(entry) => {
-				let pending_availability =
-					<inclusion::Pallet<T>>::pending_availability(entry.para_id())
-						.expect("Occupied core always has pending availability; qed");
-
-				let backed_in_number = *pending_availability.backed_in_number();
+				// Due to https://github.com/paritytech/polkadot-sdk/issues/64, using the new storage types would cause
```

Comment:
Can we extract this into a separate function that does the conversion ?

Reply:
we could to that, but I'd rather keep all the logic for this storage migration hack in the same place as it's easier to comprehend and remove in the future IMO

Reply:
Yeah, but it makes this a longer lump of code which is not really ideal for readability, but to not forget to remove it later, we should add an issue to the board.

Reply:
don't think we need an issue to track removal. When the migration will be removed, this bit of code will fail to compile, which should notify us to remove it